dockerhub-readme/generated-readmes/ngc-rapidsai-clx-dev.md

```diff
@@ -0,0 +1,205 @@
+
+
+
+## What is RAPIDS?
+
+Visit [rapids.ai](https://rapids.ai) for more information.
+
+The RAPIDS suite of software libraries gives you the freedom to execute end-to-end data science and analytics pipelines entirely on GPUs. It relies on NVIDIA® CUDA® primitives for low-level compute optimization, but exposes GPU parallelism and high-bandwidth memory speed through user-friendly Python interfaces.
+
+**NOTE:** Review our prerequisites section below to ensure your system meets the minimum requirements for RAPIDS.
+
+
+### Current Version - RAPIDS v22.04
+
+Versions of libraries included in the `22.04` images:
+- `cuDF` [v22.04](https://github.com/rapidsai/cudf/tree/v22.04.00), `cuML` [v22.04](https://github.com/rapidsai/cuml/tree/v22.04.00), `cuGraph` [v22.04](https://github.com/rapidsai/cugraph/tree/v22.04.00), `RMM` [v22.04](https://github.com/rapidsai/RMM/tree/v22.04.00), `RAFT` [v22.04](https://github.com/rapidsai/raft/tree/v22.04.00), `cuSpatial` [v22.04](https://github.com/rapidsai/cuspatial/tree/v22.04.00), `cuSignal` [v22.04](https://github.com/rapidsai/cusignal/tree/v22.04.00), `cuxfilter` [v22.04](https://github.com/rapidsai/cuxfilter/tree/v22.04.00), `clx` [v22.04](https://github.com/rapidsai/clx/tree/v22.04.00)
+
+
+### Image Types
+
+The RAPIDS images are based on [nvidia/cuda](https://hub.docker.com/r/nvidia/cuda), and are intended to be drop-in replacements for the corresponding CUDA
+images in order to make it easy to add RAPIDS libraries while maintaining support for existing CUDA applications.
+
+RAPIDS images come in three types, distributed in two different repos:
+
+The [rapidsai/rapidsai-clx](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/rapidsai/containers/rapidsai-clx/tags) repo contains the following:
+
+- `base` - contains a RAPIDS environment ready for use.
+  - **TIP: Use this image if you want to use RAPIDS as a part of your pipeline.**
+- `runtime` - extends the `base` image by adding a notebook server and example notebooks.
+  - **TIP: Use this image if you want to explore RAPIDS through notebooks and examples.**
+
+This repo (rapidsai/rapidsai-clx-dev), contains the following:
+- `devel` - contains the full RAPIDS source tree, pre-built with all artifacts in place, and the compiler toolchain, the debugging tools, the headers and the static libraries for RAPIDS development.
+  - **TIP: Use this image to develop RAPIDS from source.**
+
+### Image Tag Naming Scheme
+
+The tag naming scheme for RAPIDS images incorporates key platform details into the tag as shown below:
+```
+22.04-cuda11.0-devel-ubuntu18.04-py3.8
+ ^       ^    ^        ^         ^
+ |       |    type     |         python version
+ |       |             |
+ |       cuda version  |
+ |                     |
+ RAPIDS version        linux version
+```
+
+
+## Prerequisites
+
+- NVIDIA Pascal™ GPU architecture or better
+- CUDA [10.1/10.2/11.0/11.2](https://developer.nvidia.com/cuda-downloads) with a compatible NVIDIA driver
```

Comment:
Unrelated to your changes, but we should probably update these CUDA versions while we're touching these files.